phoenix-core/src/main/java/org/apache/phoenix/mapreduce/util/PhoenixConfigurationUtil.java

```diff
@@ -157,6 +156,8 @@
 
     public static final String MAPREDUCE_TENANT_ID = "phoenix.mapreduce.tenantid";
 
+    public static final String CUSTOM_TEST = "phoenix.mr.custom.test";
```

Comment:
Why is this phoenix.mr instead of phoenix.mapreduce?

Reply:
The majority of the properties start with "phoenix.mr" except for a couple of them as phoenix.mapreduce.